### PR TITLE
Updates docs for v0.3.1

### PR DESCRIPTION
**Pull request by saurav-c**

### CODE REVIEW

File: api-reference/sdk-reference/package-aqueduct/README.md [hunk 1]
```diff
@@ -4,7 +4,6 @@
 * [`error`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.error)
 * [`flow`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.flow)
 * [`schedule`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.schedule)
-* [`llm_op`](https://docs.aqueducthq.com/api-reference/sdk-reference/package-aqueduct/aqueduct.llm_op)
```
Comment (cw75) [inline api-reference/sdk-reference/package-aqueduct/README.md:7]: @saurav-c could you add this line back, thx!

**Review by kenxu95 — CHANGES_REQUESTED** [review]
Blocking on adding LLM Reference Package "aqueduct-llm-reference" as a top level section to `api-reference/`

**Comment by cw75** [comment]
@kenxu95 I already added that to Gitbook last night. I'll make a PR later today to update our doc generation script to auto-generate this in the future.

**Review by kenxu95 — APPROVED** [review]
ok, then I'm good then

**Pull request merged**